build.sh

```diff
@@ -90,7 +112,7 @@ popd
 
 echo "Building webrtc ..."
 pushd $WEBRTC_SRC
-gn gen out/$CONFIG --args="target_os=\"$TARGET_OS\" target_cpu=\"$TARGET_CPU\" is_debug=false use_custom_libcxx=false" || exit 1
+gn gen out/$CONFIG --args="target_os=\"$TARGET_OS\" is_debug=false use_custom_libcxx=false" || exit 1
```

Comment:
Can you explain this change?

Reply:
its from the https://trac.torproject.org/projects/tor/ticket/28205 thread, I thought I was reseting back to the original.  I'll remove it.